tiledb/sm/query/update_value.cc

```diff
@@ -79,17 +79,17 @@ UpdateValue::UpdateValue(UpdateValue&& rhs)
 UpdateValue::~UpdateValue() {
 }
 
-Status UpdateValue::check(const ArraySchema& array_schema) const {
+void UpdateValue::check(const ArraySchema& array_schema) const {
   const uint64_t update_value_size = update_value_data_.size();
 
   // Ensure field name exists.
   if (!array_schema.is_field(field_name_)) {
-    return Status_UpdateValueError("Field name doesn't exist");
+    throw StatusException(Status_UpdateValueError("Field name doesn't exist"));
```

Comment:
NIT: If you're up for it, remove Status_UpdateValueError and add a UpdateValueStatusException class to this file (example: FragmentMetadataStatusException).

We are converting all code as we go from status error to exceptions.

Reply:
Done. 
Thanks for the suggestion!